src/test/java/gov/healthit/chpl/aqa/stepDefinitions/Base.java

```diff
@@ -61,4 +61,28 @@ public WebDriverWait getWait() {
     public void setWait(final WebDriverWait wait) {
         this.wait = wait;
     }
+
+    /**
+     * Check whether the file is downloaded.
+     * @param fileName expected File name
+     */
+    public void downloadFile(final String fileName) {
```

Comment:
The name of the method does not match what the method does, nor what the documentation says